appveyor.yml

```diff
@@ -1,3 +1,5 @@
+skip_branch_with_pr: true
```

Comment:
I didn't find a "Skip branch" on Travis to avoid duplication, but I could find the following snippet that enables a "whitelist". So, Travis will just create jobs for master and Pull Requests to master.
```
branches:
  only:
    - "master"
```

[Source](https://github.com/travis-ci/travis-ci/issues/1147#issuecomment-160820262)